python/setup.py

```diff
@@ -19,7 +19,7 @@
 
 setup(
     name="py-iceberg",
-    install_requires=[],
+    install_requires=["mmh3", "numpy"],
```

Comment:
seems we just need float32 from numpy and wondering if there is a way without introducing numpy dependency?